test/test_01_hw_11.py

```diff
@@ -0,0 +1,17 @@
+from hw.hw_11_task_01 import SimplifiedEnum
+
+
+class ColorsEnum(metaclass=SimplifiedEnum):
+    __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+
+
+def test_enum():
+    assert ColorsEnum.RED == "RED"
```

Comment:
Why did you choose red only?

_I see a red door
And I want it painted black
No colors anymore
I want them to turn black._